src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -944,21 +936,23 @@ func (c controller) waitForCSV(ctx context.Context, waitTimeout time.Duration) e
 	return utils.WaitForPredicateWithContext(ctx, waitTimeout, GeneralWaitInterval, areOLMOperatorsAvailable)
 }
 
-// validateConsoleAvailability checks if the console operator is available
-func (c controller) validateConsoleAvailability() bool {
-	return c.isOperatorAvailable(NewClusterOperatorHandler(c.kc, consoleOperatorName))
-}
-
-// waitingForClusterVersion checks the Cluster Version Operator availability in the
-// new OCP cluster. A success would be announced only when the service acknowledges
-// the CVO availability, in order to avoid unsycned scenarios.
-// In case cvo changes it message we will update timer but we want to have maximum timeout
-// for this context with timeout is used
-func (c controller) waitingForClusterVersion(ctx context.Context) error {
+// waitingForClusterOperators checks Console operator and the Cluster Version Operator availability in the
+// new OCP cluster in parallel.
+// A success would be announced only when the service acknowledges the operators availability,
+// in order to avoid unsycned scenarios.
+func (c controller) waitingForClusterOperators(ctx context.Context) error {
+	// In case cvo changes it message we will update timer but we want to have maximum timeout
+	// for this context with timeout is used
 	ctxWithTimeout, cancel := context.WithTimeout(ctx, CVOMaxTimeout)
 	defer cancel()
 	isClusterVersionAvailable := func(timer *time.Timer) bool {
```

Comment:
lets change the name cause in case c.WaitForClusterVersion is false we will wait only for console